sympy/physics/vector/functions.py

```diff
@@ -280,6 +280,8 @@ def kinematic_equations(speeds, coords, rot_type, rot_order=''):
         w1, w2, w3 = speeds
```

Comment:
Could you move `w1, w2...` and the test `if w1 == w2...` to appear before `q1, q2, ...`?

Surround `==` with a single space.

`return [S.Zero]*3` since it's usually good form to return SymPy objects instead of Python objects.

Reply:
sure.
How to edit the current PR?